packages/server/test/unit/config_spec.js

```diff
@@ -1569,23 +1561,25 @@ describe('lib/config', () => {
 
         return config.mergeDefaults(obj, options)
         .then((cfg) => {
+          ['platform', 'arch'].forEach((x) => {
+            expect(cfg.resolved[x]).to.exist
```

Comment:
I just realised we could potentially do

```js
expect(cfg.resolved.arch).to.eq({ value: os.arch(), from: 'default' })
```

Or something - thoughts? We'd still delete after, so the snapshot is system agnostic.